ffmpeg/transcoder.c

```diff
@@ -193,7 +194,9 @@ int transcode(struct transcode_thread *h,
     if (ret == AVERROR_EOF) break;
                             // Bail out on streams that appear to be broken
     else if (lpms_ERR_PACKET_ONLY == ret) ; // keep going for stream copy
-    else if (ret < 0) LPMS_ERR(transcode_cleanup, "Could not decode; stopping");
+    else if (lpms_ERR_INPUT_NOKF == ret) {
+      LPMS_ERR(transcode_cleanup, "Could not decode; No keyframes found in the input");
```

Comment:
Nit: Should the second part of this error string just exactly match the description for `lpms_ERR_INPUT_NOKF` for consistency?